statsmodels/distributions/empirical_distribution.py

```diff
@@ -144,6 +144,54 @@ def __init__(self, x, side='right'):
         # which would have raised a NameError if hit, so would need to be
         # fixed.  See GH#5701.
 
+class ECDFDiscrete(StepFunction):
+    """
+    Return the Empirical Weighted CDF of an array as a step function.
+
+    Parameters
+    ----------
+    x : array_like
+        Observations
+    freq_weights : array_like
+        Weights of the observations.  sum(freq_weights) is interpreted as nobs for confint
+    side : {'left', 'right'}, optional
+        Default is 'right'. Defines the shape of the intervals constituting the
+        steps. 'right' correspond to [a, b) intervals and 'left' to (a, b].
+
+    Returns
+    -------
+    Weighted ECDF as a step function.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from statsmodels.distributions.empirical_distribution import ECDFDiscrete
+    >>>
+    >>> ewcdf = ECDFDiscrete([3, 3, 1, 4], [1, 1, 1, 1])  # For equal weights ewcdf result is same as ecdf (see above)
```

Comment:
use example that illustrate proper use, not cases that leave duplicate x values e.g.
`ewcdf = ECDFDiscrete([3, 3, 1, 4])`

and example below
`ewcdf = ECDFDiscrete([3, 1, 4], [1.25, 2.5, 5])`